src/components/button-switcher/ButtonSwitcher.tsx

```diff
@@ -57,13 +57,7 @@ export const ButtonSwitcher = ({
     </Button>
   );
   const renderSecondaryButton = (option: ButtonSwitcherOption, radiuses: { [key: string]: string | number }) => (
-    <Button
-      kind={ButtonKind.secondary}
-      rootOverrides={radiuses}
-      onClick={() => onClick(option)}
-      type="button"
-      isNotModalButton
-    >
+    <Button kind={ButtonKind.secondary} rootOverrides={radiuses} onClick={() => onClick(option)} type="button">
```

Comment:
```suggestion
    <Button kind={ButtonKind.secondary} rootOverrides={radiuses} onClick={() => onClick(option)} >
```
Type can be removed. Is the default

Reply:
Yeah, no worries I have already created this PR https://github.com/TimeChimp/fe-next/pull/1779/files
It is waiting for a new version of tacugama